js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -11,18 +11,20 @@ import { TurnContext } from 'botbuilder';
 import EventEmitter from 'events';
 import { ClientOptions, AzureOpenAI, OpenAI } from 'openai';
 import {
-    ChatCompletionCreateParams,
+    type ChatCompletionCreateParams,
```

Comment:
just curious, why does this type use the type import prefix bot not the others?

Reply:
I'm really not sure, for whatever reason if I try to do just `ChatCompletionCreateParams` without `type`, I get an error about using a namespace as a type. If you know of a better way to do this please lmk